Elevel/Application/Features/QuestionCommands/GetQuestionDetailQuery.cs

```diff
@@ -38,21 +38,45 @@ public async Task<Response> Handle(Request request, CancellationToken cancellati
 
                 if (question == null)
                 {
-                    return null;
+                    throw new NotFoundException($"Question with id {request.Id}");
                 }
 
+                var answerList = await _context.Answers.AsNoTracking()
+                    .ProjectTo<AnswerDto>(_mapper.ConfigurationProvider)
+                    .Where(x => x.QuestionId == question.Id).ToListAsync();
```

Comment:
use Include for reaching answers

Reply:
with ProjectTo it will automatically include, if you specify this in mapper profile